### DISCUSSION TITLE
BlockList / DisabledList - How it works?

### DISCUSSION THREAD

**Discussion by Cris--** [ALTaleX531/TranslucentFlyouts#124] [Q&A]
Hi,

what is exactly the difference between BlockList / DisabledList and how add a process?

```
HKEY_CURRENT_USER\Software\TranslucentFlyouts\BlockList
HKEY_CURRENT_USER\Software\TranslucentFlyouts\DisabledList
```

DWORD 32bit > process.exe (or ID?) > 1 (for enabling block/disable?)

Or a different type of registry entry?

**Comment by ALTaleX531** [comment]
https://github.com/ALTaleX531/TranslucentFlyouts/discussions/111#discussioncomment-8760640
